.ci/run_tests.sh

```diff
@@ -8,10 +8,17 @@ function red() {
     echo -e "\033[1;31m$*\033[0m"
 }
 
-export BUSTED_ARGS="-o htest -v --exclude-tags=flaky,ipv6"
+export BUSTED_ARGS="--no-k -o htest  -v --exclude-tags=flaky,ipv6"
 
 if [ "$KONG_TEST_DATABASE" == "postgres" ]; then
     export TEST_CMD="bin/busted $BUSTED_ARGS,cassandra,off"
+    psql -v ON_ERROR_STOP=1 -h ${KONG_TEST_PG_HOST} -U ${KONG_TEST_PG_USER} -d ${KONG_TEST_PG_DATABASE} <<-EOSQL
```

Comment:
Should this be added in the KBT Postgres container instead? Adding it here is really not friendly for development environment as people often run `busted` directly on a single test file and not necessarily uses the `.ci/run_tests.sh` for running the tests.

Reply:
I feel like there are pros and cons of both and this is something I debated and even roughed out both ways of doing it.

One requirement is it needs to happen locally when folks run tests as well as travis-ci, Jenkins and Github Actions. I couldn't see a way to place it in KBT that would require duplicating the logic somewhere else as well (?)